one-node-ce/README.md

```diff
@@ -91,11 +95,13 @@ After `Dockerfile_<distro>` installs the RPM or DEB file, it runs `tools/cleanup
 
 # Testing with ./run_tests.sh
 
+Once you have built your container, you can test it using the `./run_tests.sh` script (or by running `make test`).
+
 The `./run_tests.sh` script verifies that the container can execute Vertica and some of the additional libraries. This script requires a [local copy of the vsql client](#getting-a-local-copy-of-vsql). You can run the script directly, or you can run it as `make test`.
```

Comment:
The last sentence in line 100 is made redundant by the addition at line 98.  Maybe consider removing it?

Reply:
Good point.  Thanks